src/main/target/system_stm32h7xx.c

```diff
@@ -546,7 +546,11 @@ void MPU_Config()
     // The Base Address 0x24000000 is the SRAM1 accessible by the SDIO internal DMA.
     MPU_InitStruct.Enable = MPU_REGION_ENABLE;
     MPU_InitStruct.BaseAddress = 0x24000000;
+#if defined(EXST) || defined(BOOTLOADER)
```

Comment:
Where is `BOOTLOADER` used? And it should really be `USE_EXST`to match the naming convention for binary type defines.

Reply:
BOOTLOADER should be removed, @jflyper kept merged that accidentally, I'll fix that.